launch-or-update-local.ps1

```diff
@@ -32,7 +32,37 @@ param(
         [switch]$u=$true
     )
 
+# Parameters   
 $chart="src/buyingcatalogue"
+$namespace="buyingcatalogue"
+
+# Check context is docker dashboard
+$context = kubectl config current-context
+if ($context -ne "docker-desktop")
+{
+    write-host "Not running in the local context - please switch to docker desktop"
+    write-host "Exiting...."
+    start-sleep 20
+    exit 1
+}
+
+# Add namespace if missing
+$namespacePresent = kubectl get namespace $namespace
+if (!($namespacePresent))
+{
+    kubectl create namespace $namespace
+    write-host "`nNamespace has been created`n"
+}
+
+# Check for Azure repo creds
+$regCredentials=kubectl get secret regcredlocal -n $namespace
+if (!($regCredentials))
+{
+    write-host "Missing Credentials for ACR - please follow instructions in docs\k8s-private-registry.md"
+    write-host "Exiting...."
+    start-sleep 20
```

Comment:
What benefit does this sleep bring here?

Reply:
It lets you see the error message before PowerShell exits and closes - if launched directly

Reply:
ahh, seems sensible then